src/PrestaShopBundle/Translation/PrestaShopTranslatorTrait.php

```diff
@@ -50,9 +50,9 @@ trait PrestaShopTranslatorTrait
     public function trans($id, array $parameters = [], $domain = null, $locale = null)
     {
         if (isset($parameters['legacy'])) {
-            $legacy = $parameters['legacy'];
             unset($parameters['legacy']);
```

Comment:
We could add a deprecation message here as well, warning that it's no longer supported